secret-sharing/src/vss/fuzz/main.rs

```diff
@@ -0,0 +1,89 @@
+use group::ff::PrimeField;
+use honggfuzz::fuzz;
+use rand::{rngs::StdRng, Rng, SeedableRng};
+
+use secret_sharing::vss::{matrix::VerificationMatrix, polynomial::BivariatePolynomial};
+
+fn main() {
+    loop {
+        fuzz!(|data: &[u8]| {
+            fuzz_bivariate_polynomial_random(data);
+            fuzz_bivariate_polynomial_from_seed(data);
+            fuzz_bivariate_polynomial_from_bytes(data);
+
+            fuzz_verification_matrix_random(data);
+            fuzz_verification_matrix_from_seed(data);
```

Comment:
This probably makes it a bit awkward for the fuzzer as the same data is used for different code paths. Ideally you would have one case for each of these or split the data up.